src/signals/incident/definitions/wizard-step-3-contact.js

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2018 - 2021 Gemeente Amsterdam
+import { Validators } from 'react-reactive-form'
+
+import configuration from 'shared/services/configuration/configuration'
+import { validatePhoneNumber } from '../services/custom-validators'
+import IncidentNavigation from '../components/IncidentNavigation'
+import FormComponents from '../components/form'
+
+export default {
+  label: 'Contactgegevens',
+  nextButtonLabel: 'Volgende',
+  nextButtonClass: 'action primary arrow-right',
+  previousButtonLabel: 'Vorige',
+  previousButtonClass: 'action startagain',
+  formAction: 'UPDATE_INCIDENT',
+  form: {
+    controls: {
+      phone: {
+        meta: {
+          autoComplete: 'tel',
+          autoRemove: /[^\d ()+-]/g,
+          heading: 'Mogen we u bellen voor vragen?',
+          label: 'Wat is uw telefoonnummer?',
+          path: 'reporter.phone',
+          subtitle:
+            'Zo kunt u ons helpen het probleem sneller of beter op te lossen.',
+          type: 'tel',
+          wrap: FormComponents.TextInput,
+        },
+        render: FormComponents.WithHeading,
+        options: {
```

Comment:
I would prefer to see the actual component here, so in this case TextInput, and then have withHeading as a meta property. However, I understand that if the preference is to have a separate component that can be used for more form components that this is the pragmatic solution.

Reply:
That's not possible with the current setup; `react-reactive-form` expect a component to render. It cannot be reversed.